shell/browser/browser_mac.mm

```diff
@@ -377,11 +377,22 @@ void RemoveFromLoginItems() {
   NSDictionary* options = DictionaryValueToNSDictionary(about_panel_options_);
 
   // Credits must be a NSAttributedString instead of NSString
-  id credits = options[@"Credits"];
+  NSString* credits = (NSString*)options[@"Credits"];
   if (credits != nil) {
+    // Check if app is running in dark mode
+    NSString* mode = [[NSUserDefaults standardUserDefaults]
+        stringForKey:@"AppleInterfaceStyle"];
+    BOOL isDarkMode = [mode isEqualToString:@"Dark"];
+
+    // Set color of credits depending on if we're in dark mode or not.
+    NSColor* color = isDarkMode ? [NSColor whiteColor] : [NSColor blackColor];
```

Comment:
It would be more correct to use `NSColor.textColor` here. It also has the benefit of automatically adapting to dark mode.

Reply:
thanks for the tip, i updated it 🙇‍♀